src/databricks_clean_room_orchestrator/client.py

```diff
@@ -273,6 +273,9 @@ def _prepareAndRunNotebookHelper(
       if (response["workspace_status"] != "PROVISIONING"):
         raise RuntimeError("Workspace could not be provisioned")
       time.sleep(10)
+    
+    print("Finalize station workspace setup")
+    self._rest_client.setupStationResource(self._clean_room, self._station_name, Resource.WORKSPACE)
```

Comment:
Question: We seem to do the same step at line 267. So why is this call needed again? Consider adding a comment.